godot-cell/src/borrow_state.rs

```diff
@@ -60,6 +66,15 @@ impl BorrowState {
         self.shared_count
     }
 
+    /// Returns the number of tracked shared references in the current thread.
+    #[cfg(feature = "threads")]
+    pub fn thread_shared_count(&self) -> usize {
+        self.thread_shared_count
+            .get(&std::thread::current().id().to_owned())
+            .unwrap_or(&0usize)
+            .to_owned()
```

Comment:
`to_owned()` on a `&usize` is... unconventional 😀 
Just use `*` in front. 

Please also remove the one after `id()`, this obfuscates code significantly otherwise.